src/forms/redux-form/Checkbox.jsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import Checkbox from '../Checkbox';
+
+const ReduxFormCheckbox = props => {
+	const {
+		meta, // eslint-disable-line no-unused-vars
+		input,
+		...other
+	} = props;
+
+	return <Checkbox checked={input.value === true} {...input} {...other} />;
```

Comment:
any chance `input` will ever be undefined or null?

Reply:
input should always be defined from `redux-form`